docs/extendedusage.rst

```diff
@@ -198,6 +200,23 @@ Traits
     'The index of the first track to return'
 
 
+RAML1.0 Types
+~~~~~~~~~~~~~~
+
+.. code-block:: python
+
+    >>> api.types
+    {'Person': ObjectType(name='Person', properties={'name': Property(type='string')})}
+    >>> person = api.types['Person']
+    >>> person.type
+    'object'
+    >>> person.description
+    'a Person is a type describing human beings'
+    >>> person.properties
+    {'name': Property(type='string')})
```

Comment:
as everywhere else, types are referred by name, I think it is better to store it as a dict, which has fast and easy 'byName' access. I would have the same comment for ResourceType implementation, which is afaik a list